Readme.md

```diff
@@ -343,6 +343,14 @@ const levelPrettifier = logLevel => `LEVEL: ${levelColorize(logLevel)}`
 }
 ```
 
+In addition to this, if / end statement blocks can also be specified. Else statements and nested conditions are not supported.
+
+```js
+{
+  messageFormat: '{levelLabel} - {if pid}{pid}{end} - url:{req.url}'
```

Comment:
```suggestion
  messageFormat: '{levelLabel} - {if pid}{pid} - {end}url:{req.url}'
```

Reply:
Agreed. This would be a better example to showcase the condition option.